spec.html

```diff
@@ -6913,9 +6903,11 @@ <h1>Static Semantics: ContainsDuplicateLabels</h1>
           ThrowStatement
           DebuggerStatement
 
-        Block : `{` `}`
+        Block :
+          `{` `}`
```

Comment:
Elsewhere, single alternative productions aren't multiline. I think I'd prefer single-alternative to remain single line. WDYT?

Reply:
Oh, maybe this is not an issue since it appears as multiline in source only and will render as single line.

Reply:
No, it'll be two. I'll revert this change for the shorter ones.